webapp/components/CategoriesSelect/CategoriesSelect.spec.js

```diff
@@ -35,14 +37,19 @@ describe('CategoriesSelect.vue', () => {
     },
   ]
   beforeEach(() => {
+    provide = {
+      $parentForm: {
+        update: jest.fn(),
+      },
+    }
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-08T15:35:27Z" title="Friday, November 8th 2019, 4:35:27 pm +01:00">Nov 8, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -35,14 +37,19 @@ describe('CategoriesSelect.vue', () => {
     },
   ]
   beforeEach(() => {
+    provide = {
+      $parentForm: {
+        update: jest.fn(),
+      },
+    }
```


This is how we can implement dedicated form input components, e.g. the `CategoriesSelect` component. @mattwr18

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-12T20:22:11Z" title="Tuesday, November 12th 2019, 9:22:11 pm +01:00">Nov 12, 2019</time>_

---


looks pretty cool @roschaefer